src/kubernetes/api_watcher.rs

```diff
@@ -210,13 +210,13 @@ mod tests {
 
     use super::*;
     use futures::StreamExt;
-    use httpmock::MockServer;
-    use httpmock::{Method::GET, Then, When};
     use k8s_openapi::WatchOptional;
     use k8s_openapi::{
         api::core::v1::Pod,
         apimachinery::pkg::apis::meta::v1::{ObjectMeta, Status, WatchEvent},
     };
+    use wiremock::matchers::{method, path};
+    use wiremock::{Mock, MockServer, ResponseTemplate};
```

Comment:
I'm not opposed to switching to `wiremock`, but was there a particular reason that motivated it? The usage below looks pretty much the same as the previously used `httpmock`.

Reply:
Aha, never mind, I missed your comment above. 👍 to the change.